.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I'm a bit confused as to what bug would refer to in the context of our tasks repository. If there's a bug on one of our actual software projects, the issue should probably reside there.

I could see the bug analogy being used in the case of tasks as we are doing something wrong. Maybe some simple rephrasing of these would work? I'll suggest some changes for this.